django/db/migrations/writer.py

```diff
@@ -169,14 +169,15 @@ def as_string(self):
                 imports.remove(line)
                 self.needs_manual_porting = True
         imports.discard("from django.db import models")
-        items["imports"] = "\n".join(imports) + "\n" if imports else ""
+        sorted_imports = sorted(imports, key=lambda i: i.split()[1])
```

Comment:
could you add a comment explaining the ordering we are trying to achieve? `i.split()[1]` isn't immediately obvious to me.